### PR TITLE
Add Kubevirt/OSV VM events

### PR DESCRIPTION
**Pull request by nasark**
Add policy event definitions for Kubevirt and Openshift Virtualization

Depends on:
- [x] https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/256
- [x] https://github.com/ManageIQ/manageiq-providers-openshift/pull/273
 
@miq-bot assign @agrare
@miq-bot add_label enhancement
@miq-bot add_reviewer @Fryguy, @agrare

### CODE REVIEW

File: content/automate/ManageIQ/System/Event/EmsEvent/KUBEVIRT.class/__class__.yaml [hunk 1]
```diff
@@ -0,0 +1,433 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: 
+    display_name: 
+    name: Kubernetes
```
Comment (agrare) [inline content/automate/ManageIQ/System/Event/EmsEvent/KUBEVIRT.class/__class__.yaml:8]: Should this be Kubevirt?

File: content/automate/ManageIQ/System/Event/EmsEvent/OPENSHIFT_INFRA.class/__class__.yaml [hunk 2]
```diff
@@ -0,0 +1,433 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: 
+    display_name: 
+    name: Kubernetes
```
Comment (agrare) [inline content/automate/ManageIQ/System/Event/EmsEvent/OPENSHIFT_INFRA.class/__class__.yaml:8]: Similarly, should this be Openshift Virtualization?

File: content/automate/ManageIQ/System/Event/EmsEvent/KUBEVIRT.class/__class__.yaml (hunk 1 above)
Comment (nasark) [reply content/automate/ManageIQ/System/Event/EmsEvent/KUBEVIRT.class/__class__.yaml]: yes, updated to match the class name

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit 2871be3dd8cd35d477324865d31da38b3717e5c8.

```text
commit 2871be3dd8cd35d477324865d31da38b3717e5c8
Author: Adam Grare <adam@grare.com>
Date:   Fri Nov 15 14:36:39 2024 -0500

    Merge pull request #755 from nasark/kubevirt_osv_vm_events
    
    Add Kubevirt/OSV VM events
    
    (cherry picked from commit 4f2deec46e67001dc589aef3c9cb04cedb67fab0)
```

**Pull request merged**